ftw/simplelayout/browser/resources/RequestTracker.js

```diff
@@ -98,10 +98,12 @@
       Disable kss spinner. In order for this to work, this file must be evaluated after kss-bbb.js.
       As you can see in the lib profile's jsregistry.xml.
     */
-    $.ajaxSetup({ global: false });
-
-    $document.on("ajaxSend", function(event, jqxhr, params) { track(params.url); });
-    $document.on("ajaxComplete", function(event, jqxhr, params) { untrack(params.url); });
+    $(function() {
+      $document.unbind("ajaxStart");
+      $document.unbind("ajaxStop");
+      $document.on("ajaxSend", function(event, jqxhr, params) { track(params.url); });
+      $document.on("ajaxComplete", function(event, jqxhr, params) { untrack(params.url); });
+    })
```

Comment:
If I understand this correctly: you are defining a new function which is not referenced in a variable nor called after definition. When do you expect this code to be executed?